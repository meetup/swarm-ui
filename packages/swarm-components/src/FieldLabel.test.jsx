import React from 'react';
import { shallow } from 'enzyme';
import FieldLabel from './FieldLabel';

describe('FieldLabel', () => {
	const testCases = [['Regular', <FieldLabel key="1">Field Label</FieldLabel>]];

	test.each(testCases)('Snapshot: %s', (description, element) => {
		expect(shallow(element)).toMatchSnapshot();
	});
});
