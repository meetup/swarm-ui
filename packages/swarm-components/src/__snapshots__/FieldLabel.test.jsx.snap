// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldLabel Snapshot: Regular 1`] = `
<label
  data-swarm-field-label={true}
>
  Field Label
</label>
`;
